cmd/build/main.go

```diff
@@ -16,20 +16,36 @@
 package main
 
 import (
+	"encoding/json"
 	"flag"
 	"log"
+	"os"
 
 	"github.com/project-oak/transparent-release/build"
 )
 
 func main() {
-	buildConfigPathPtr := flag.String("config", "",
+	buildConfigPath := flag.String("build_config_path", "",
 		"Required - Path to a toml file containing the build configs.")
```

Comment:
nit & pre-existing: "a" toml file vs -> "the" toml file --- would we expect there to be more than one toml file in general? Maybe also want to clarify that these are the build configs to build the gitRootDir? Is that correct?